### PR TITLE
add documentation for date and datetime validation

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited, thank you!

File: app/views/pages/tutorials/forms/validation.liquid [hunk 1]
```diff
@@ -197,6 +197,58 @@ Available options are:
...
+### datetime
+
+```yaml
+my_date:
+  validation:
+    datetime:
+      after: "2 days ago"
+      message: "Must be not earlier than 2 days ago"
+
+my_date:
```
Comment (pavelloz) [inline app/views/pages/tutorials/forms/validation.liquid]:
```suggestion
my_date2:
```

File: app/views/pages/tutorials/forms/validation.liquid [hunk 2]
```diff
@@ -197,6 +197,58 @@ Available options are:
 | odd | Input must be an odd number if set to true | must be odd |
 | even | Input must be an even number if set to true | must be even |
 
+### datetime
+
+```yaml
+my_date:
+  validation:
+    datetime:
+      after: "2 days ago"
```
Comment (pavelloz) [inline app/views/pages/tutorials/forms/validation.liquid]:
```suggestion
      after: "10 minutes ago"
```

File: app/views/pages/tutorials/forms/validation.liquid [hunk 3]
```diff
@@ -197,6 +197,58 @@ Available options are:
...
+      message: "Must be not earlier than 2 days ago"
```
Comment (pavelloz) [inline app/views/pages/tutorials/forms/validation.liquid]:
```suggestion
      message: "Must be not earlier than 10 minutes ago"
```

File: app/views/pages/tutorials/forms/validation.liquid [hunk 4]
```diff
@@ -197,6 +197,58 @@ Available options are:
...
+    datetime:
+      before: "now"
+      message: "Must be in the past"
+```
+
+The possible date constraint options are:
+
+| Validation | Description |
+|-----------|-------------|
+| is_at | Must be at this exact datetime |
```
Comment (pavelloz) [inline app/views/pages/tutorials/forms/validation.liquid]:
```suggestion
| is_at | Must be at the same time, to the second |
```

File: app/views/pages/tutorials/forms/validation.liquid [hunk 5]
```diff
@@ -197,6 +197,58 @@ Available options are:
 | odd | Input must be an odd number if set to true | must be odd |
 | even | Input must be an even number if set to true | must be even |
 
+### datetime
+
```
Comment (pavelloz) [inline app/views/pages/tutorials/forms/validation.liquid:202]:
```suggestion
Datetime is a date with exact time, to the second. For example: `Fri Feb 07 15:29:43 2020 UTC`.

 Our system can convert human phrases (for example: `1 hour ago`, `3 hours from now`, `now`, `1 day ago`) to datetime, so you can write them easier.
 
Note: `yesterday` will take yesterdays date at midnight, if you want to preserve time of the day, use `1 day ago`
```

File: app/views/pages/tutorials/forms/validation.liquid [hunk 6]
```diff
@@ -197,6 +197,58 @@ Available options are:
...
+    datetime:
+      before: "now"
+      message: "Must be in the past"
+```
+
+The possible date constraint options are:
+
+| Validation | Description |
+|-----------|-------------|
+| is_at | Must be at this exact datetime |
```
Comment (Slashek) [inline app/views/pages/tutorials/forms/validation.liquid]:
```suggestion
| at | Must be at this exact datetime |
```

File: app/views/pages/tutorials/forms/validation.liquid [hunk 7]
```diff
@@ -197,6 +197,62 @@ Available options are:
...
+
+ Our system can convert human phrases (for example: `1 hour ago`, `3 hours from now`, `now`, `1 day ago`) to datetime, so you can write them easier.
+ 
+Note: `yesterday` will take yesterdays date at midnight, if you want to preserve time of the day, use `1 day ago`
+```yaml
+my_date:
+  validation:
+    datetime:
+      after: "10 minutes ago"
+      message: "Must be not earlier than 10 minutes ago"
```
Comment (pavelloz) [inline app/views/pages/tutorials/forms/validation.liquid]:
```suggestion
      message: "Must not be earlier than 10 minutes ago"
```

**Pull request merged**